_episodes/03-transform-data.md

```diff
@@ -463,10 +464,30 @@ A quick `browse` confirms that the data is in the tidy format.
 Time to save our data. Stata 16, Stata 15, and Stata 14 share the same format, so you can just go ahead and save the data using the `save` command.
 
 ```
-save "data/WDI-select-variables.dta", replace
+save "data/WDI-select-variables.dta"
 ```
 {: .source}
 
+Load the saved data and generate a dummy variable that takes value one if the `gdp_per_capita` is higher than the mean.
+```
+gen high_gdp_per capita=(gdp_per_capita >=15453.68 ) if !missing(gdp_per_capita)
```

Comment:
@ariedamuco Please conform to style guide. `generate high_gdp_per capita = (gdp_per_capita >= 15453.68 ) if !missing(gdp_per_capita)